homework/08LessonEight/script.js

```diff
@@ -17,30 +17,30 @@ $(document).on('ready', function() {
 
   // Problem 1:
   // Get the children of #first-list using the .children() method
-  window.firstListChildren = null;
+  window.firstListChildren = $firstList.children();
 
   // Problem 2:
   // Get the children of #second-list using the .children() method
-  window.secondListChildren = null;
+  window.secondListChildren = $secondList.children();
   
   // Problem 3:
   // Now use the .detach() method on window.firstListChildren to
   // remove those elements from the document
-  window.firstListChildren;
+  window.firstListChildren.detach();
 
   // Problem 4:
   // Now use the .detach() method on window.secondListChildren to
   // remove those elements from the document
-  window.secondListChildren;
+  window.secondListChildren.detach();
 
   // Problem 5:
   // use $finalList.prepend() to put window.firstListChildren
   // at the beggining of #final-list
-  $finalList;
+  $finalList.prepend(window.firstListChildren);
 
   // Problem 6:
   // use $finalList.append() to put window.secondListChildren
   // at the end of #final-list
-  $finalList;
+  $finalList.append(window.secondListChildren);
```

Comment:
All tests passed! You're too smart for this class.

![passing](https://media.giphy.com/media/jBLJuTezU5tra/giphy.gif)